entrypoint.sh

```
@@ -35,7 +85,7 @@ if ! check_adc; then
   gcloud auth application-default login --quiet
   # Setup project and quota
   gcloud config set project ${ANTHROPIC_VERTEX_PROJECT_ID}
-  gcloud auth application-default set-quota-project ${ANTHROPIC_VERTEX_PROJECT_QUOTA}
+  gcloud auth application-default set-quota-project ${ANTHROPIC_VERTEX_QUOTA}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)